daemon/daemon.go

```diff
@@ -1501,6 +1501,15 @@ func (daemon *Daemon) newBaseContainer(id string) *container.Container {
 	return container.NewBaseContainer(id, daemon.containerRoot(id))
 }
 
+// ResolveRepository gets the repository information from a registry service.
+func (daemon *Daemon) ResolveRepository(ref reference.Named) (*registry.RepositoryInfo, error) {
+	return daemon.RegistryService.ResolveRepository(ref)
```

Comment:
can we downcase and hide RegistryService now? I don't see any other users outside of the previously existing use.